crates/bevy_ui/src/render/mod.rs

```diff
@@ -161,9 +163,18 @@ pub struct ExtractedUiNodes {
     pub uinodes: Vec<ExtractedUiNode>,
 }
 
+pub struct ExtractAtlasDetails {
```

Comment:
I think this intermediate type probably overcomplicates things and isn't need.

Reply:
I actaully added it because the tuple of the 5 elements felt overcomplicated was difficult to work with, I can revert back to not using it if other agree

Reply:
Update: Changed to remove the struct after reworking into separate system